### ISSUE TITLE
Bug: unit-no-unknown raise warning on hexa color

### ISSUE THREAD

**Issue by mathilde-pellerin** [stylelint/stylelint#1303]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

`unit-no-unknown` raise warning on hexa color.

> Which rule, if any, is this issue related to?

`unit-no-unknown`

> What CSS is needed to reproduce this issue?

```css
.foo {
  background-color: #106cc8;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "color-hex-case": "lower",
    "color-hex-length": "long",
    "unit-no-unknown": true
  }
}
```

> Which version of stylelint are you using?

Stylelint version: 6.4.0

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Stylelint running using PostCSS plugin for Gulp.

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

No warning on hexa colors.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:

```shell
42:21   ⚠  Unexpected unknown unit "cc8" (unit-no-unknown) [stylelint]
```

**Comment by evilebottnawi** [comment]
@mathilde-pellerin Fixed in https://github.com/stylelint/stylelint/pull/1302. Ready 6.0.1 :+1:

**Comment by mathilde-pellerin** [comment]
That was quick 👍 
Thanks!
